src/main/java/com/radixdlt/crypto/ECKeyPair.java

```diff
@@ -161,4 +201,83 @@ public String toString() {
 		return String.format("%s[%s]",
 			getClass().getSimpleName(), Bytes.toBase64String(getPublicKey().getBytes()));
 	}
+
+	// ###  From Client Library  ###
+
+	@JsonProperty("version")
+	@DsonOutput(DsonOutput.Output.ALL)
+	private short version = 100;
+
+	// Placeholder for the serializer ID
+	@JsonProperty(SerializerConstants.SERIALIZER_NAME)
+	@DsonOutput(DsonOutput.Output.ALL)
+	private SerializerDummy serializer = SerializerDummy.DUMMY;
+
+	public ECKeyPair(byte[] publicKey, byte[] privateKey) {
+		try {
+			this.publicKey = new ECPublicKey(publicKey);
+		} catch (CryptoException e) {
+			throw new IllegalArgumentException("Failed to create public key from bytes", e);
+		}
+		this.privateKey = Arrays.copyOf(privateKey, privateKey.length);
+	}
+
+	public static ECKeyPair fromFile(File file) throws CryptoException {
+		InputStream inputStream;
```

Comment:
Should use try-with-resources here to ensure stream is closed (which it isn't right now)

Reply:
Thx! @clstrfsck provided me with a simplified improved implementation that I copied over.